src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -40,7 +40,10 @@ void HBProcessor::process(const cpp2::HBReq& req) {
       onFinished();
       return;
     }
+  }
 
+  // set or check storaged's cluster id
+  if (role == cpp2::HostRole::STORAGE) {
```

Comment:
why check the role == cpp2::HostRole::STORAGE again? split the if code block for different function?

Reply:
I'll merge it